### PR TITLE
Add READMEs to all crates and clarify they are part of Bevy

### PR DESCRIPTION
**Pull request by ameknite**
# Objective

The majority of the Bevy crates don't have a README.md.

Screenshot 2023-10-01 at 20 19 33

I want to add a README so that when search for a crate on crates.io, it reports that:

- The crate is a component of the BevyEngine.
- The crate is not intended for independent use.

### CODE REVIEW

**Review by alice-i-cecile — APPROVED** [review]
A little more color about what each crate does wouldn't hurt, but I think this is an improvement.

**Comment by Vrixyz** [comment]
A readme for each crate is definitely an improvement.

I'm approving that PR, but I'd like to nitpick on potential problems:

## modular design

We should make sure the wording
> It is not intended for independent use, use it only as a part of Bevy

is not too strong for a few crates?

As `bevy_ecs` can be (and is[^1]) used without all other bevy ecosystem, adding this disclosure to the wrong crate might drive users off the carefully though out modularity.

From my understanding, the crates listed seem to be indeed not thought out for independent use (yet ?) :+1:.

## Trust authority

> Bevy Animation is a component of the [Bevy Engine](https://bevyengine.org/)

I agree it's a good information to share, but it might be a dangerous information to have in readme, because it teaches users to rely on readme information, where a malicious user could create a fake Readme saying their crate is part of bevy organisation and then do bad things™️ (phishing, malicious packages...).



[^1]: https://store.steampowered.com/app/2198150/Tiny_Glade/

**Review by Vrixyz — APPROVED** [review]

**Review by 5tr1k3r — APPROVED** [review]

**Comment by ameknite** [comment]
@Vrixyz 

"and then do bad things™️ (phishing, malicious packages...)." 
I don't think the crates.io policy allows it.
https://github.com/rust-lang/rfcs/pull/3463

**Review by alice-i-cecile — CHANGES_REQUESTED** [review]
"revise" needs to change to "review". This is better boilerplate otherwise though!

File: crates/bevy_a11y/README.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Bevy A11Y
+
+[Bevy A11Y](https://github.com/bevyengine/bevy/tree/main/crates/bevy_a11y) is a component of the [Bevy Engine](https://bevyengine.org/). You can revise the list of official crates [here](https://github.com/bevyengine/bevy/tree/main/crates).
```
Comment (mockersf) [inline crates/bevy_a11y/README.md:3]: so... `bevy` isn't an official crate?

File: crates/bevy_app/README.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# Bevy App
+
+[Bevy App](https://github.com/bevyengine/bevy/tree/main/crates/bevy_app) is a component of the [Bevy Engine](https://bevyengine.org/). You can revise the list of official crates [here](https://github.com/bevyengine/bevy/tree/main/crates).
+
+This crate is intended to be used as a part of [Bevy](https://crates.io/crates/bevy) and not independently. However, feel free to use it if it fits your needs.
```
Comment (mockersf) [inline crates/bevy_app/README.md:5]: not true for `bevy_app`, it can be used independently

Comment (alice-i-cecile) [reply crates/bevy_app/README.md:5]: Right, I guess `bevy_app` with `bevy_ecs` is perfectly valid.

File: crates/bevy_a11y/README.md (hunk 1 above)
Comment (ameknite) [reply crates/bevy_a11y/README.md]: I could change it to:
[Bevy A11Y](https://github.com/bevyengine/bevy/tree/main/crates/bevy_a11y) is a component of the [Bevy](https://crates.io/crates/bevy) crate. You can review the list of official crates [here](https://github.com/bevyengine/bevy/tree/main/crates).

that would be better?

File: crates/bevy_app/README.md (hunk 2 above)
Comment (ameknite) [reply crates/bevy_app/README.md]: Ok, I remove it

File: crates/bevy_a11y/README.md (hunk 1 above)
Comment (alice-i-cecile) [reply crates/bevy_a11y/README.md]: I like that TBH

**Review by Vrixyz — APPROVED** [review]

File: crates/bevy_a11y/README.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+# Bevy A11Y
+
+[Bevy A11Y](https://github.com/bevyengine/bevy/tree/main/crates/bevy_a11y)  is a component of the [Bevy](https://crates.io/crates/bevy) crate. You can review the list of official crates [here](https://github.com/bevyengine/bevy/tree/main/crates).
```
Comment (mockersf) [inline crates/bevy_a11y/README.md:3]:
```suggestion
[Bevy A11Y](https://github.com/bevyengine/bevy/tree/main/crates/bevy_a11y) is a component of the [Bevy](https://crates.io/crates/bevy) crate. You can review the list of official crates [here](https://github.com/bevyengine/bevy/tree/latest/crates).
```
* there is an extra space
* links on crates.io should be to the latest branch

**Comment by mockersf** [comment]
I think linking to the GitHub repo for the list of crates is wrong. To have a good list, we could add a GitHub group to the authorised publishers on crates.io, then we could link to it.

A team named official would have a link like `https://crates.io/teams/github:bevyengine: official `. It doesn't work for now as it doesn't exist. Currently all crates are listed at https://crates.io/users/cart but there are a few extra things.

**Comment by ameknite** [comment]
@mockersf Oh, that would be great, but I think Carl is the only one with the permissions. For now, I believe linking to the GitHub repo is a good temporary solution. Or should I change the link to https://crates.io/users/cart ?

**Comment by mockersf** [comment]
let's try and see if we can get that team created and added as a publisher 👍

**Comment by mockersf** [comment]
we now have https://crates.io/teams/github:bevyengine:publish as an owner of the crates on crates.io 🎉

**Pull request closed without merging**